.env

```diff
@@ -36,3 +36,15 @@ MAINNET_SEND_MANY_CONTRACT_ID=SP3FBR2AGK5H9QBDH3EEN6DF8EK8JY7RX8QJ5SVTE.send-man
 
 # Directory containing Stacks 1.0 BNS data extracted from https://storage.googleapis.com/blockstack-v1-migration-data/export-data.tar.gz
 # BNS_IMPORT_DIR=/extracted/export-data-dir/
+
+# Used to run the API without a stacks-node or postgres connection. Only the Rosetta 
+# /network/list
```

Comment:
Do you think it might be more helpful for this (and the readme) to include a link to https://www.rosetta-api.org/docs/node_deployment.html#offline-mode-endpoints rather than listing the endpoints?

Reply:
yeah, that is more elegant.  :)